themes/mCommerceTheme/styles/components/LviNutritions.json

```diff
@@ -0,0 +1,5 @@
+{
+  ".lviNutritions": {
+    "height": 55
```

Comment:
![image](https://user-images.githubusercontent.com/15328038/153384194-4c602497-c86f-4379-8e5d-ea65608b7581.png)
too much empty spaces between items. Align separator with text and remove: from header